### PR TITLE
FLEX HEADLINE

### PR DESCRIPTION
**Pull request by johnnieskywalker**
# [Implemented new flex Headline](https://app.shortcut.com/oazo-apps/story/7770/implement-new-version-of-vault-headline)
  
## Changes 👷‍♀️
- Added couple of CSS properties to make headline more flex like in design https://www.figma.com/file/XeMo7s0ZJgXgCHmOd588QY/Discover-Oasis-%235366?node-id=2904%3A16028&t=vGcRRXMM8TnPPiGp-0
- 
  
## How to test 🧪
- Here's short video that tests multiple cases and all are fine IMHO

https://user-images.githubusercontent.com/6871923/216317831-f786001c-242a-428d-9dbd-0124cbb7a217.mp4

- This is longer version with more extreme trials, mobile issues found during this recording are fixed, but you can test it too, try think of even more unusual edge cases

https://user-images.githubusercontent.com/6871923/216317995-ab57c342-7cac-4557-9965-fbaa4ac0ecf3.mp4

### CODE REVIEW

File: features/follow/common/FollowButtonControl.tsx [hunk 1]
```diff
@@ -61,6 +61,26 @@ export function FollowButtonControl({
     }
   }
   const isWalletConnected = accountIsConnectedValidator({ account: followerAddress })
+  async function unfollowVault(jwtToken: string) {
```
Comment (johnnieskywalker) [inline features/follow/common/FollowButtonControl.tsx:64]: this commit was for different branch I'll revert it however it doesn't break anything

Comment (johnnieskywalker) [reply features/follow/common/FollowButtonControl.tsx]: I reset branch and force pushed without that commit that was supposed to go to another branch

**Review by piotrkonowrocki — CHANGES_REQUESTED** [review]
Checked, tried to break it, looks like everything works as intended. I have one tiny comment - according to the designs, there should be 16px gap between two lines when they can't fit next to each other. The easiest way to achieve it would be adding `rowGap: 3` their container.

Also, I would feel more comfortable if some other front-end beside me checked it, since there might be some edge cases here that I didn't thought of.

**Comment by johnnieskywalker** [comment]
https://user-images.githubusercontent.com/6871923/216594826-3afacb10-6f92-4c2c-b711-b06e9063a17d.mp4

**Review by marcinciarka — APPROVED** [review]

**Review by piotrkonowrocki — APPROVED** [review]
